test/abstract_unit.rb

```diff
@@ -1,6 +1,11 @@
 spec_name = ENV['ADAPTER'] || 'postgresql'
 require 'bundler'
 require 'minitest/autorun'
+if spec_name == "trilogy"
+  require "activerecord-trilogy-adapter"
+  require "trilogy_adapter/connection"
+  ActiveRecord::Base.extend TrilogyAdapter::Connection
+end
```

Comment:
Let's remove this code. Loading the gems should be done in the Gemfile not here.

Is extending ActiveRecord::Base really needed? If so, then doesn't this has to happen when loading the trilogy gem in all cases (not just tests)?